sdk/python/lib/test/provider/experimental/test_analyzer.py

```diff
@@ -1067,28 +1066,6 @@ def test_analyze_component_mutually_recursive_complex_types_file():
     }
 
 
-def test_analyze_component_excluded_files():
```

Comment:
Since we now pass a list of components and not a source dir to scan, this test doesn't make sense anymore.